x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -623,7 +623,8 @@ Provider. This will automatically configure many of the integration options
 between the IdP and the SP.
 
 The Elastic Stack supports generating such a metadata file using the
-`bin/elasticsearch-saml-metadata` command in your {es} directory.
+`bin/elasticsearch-saml-metadata` command in your {es} directory or <<security-api-saml-sp-metadata,SAML service
+provider metadata API>>.
```

Comment:
The sentence below reads strange now that we have two ways of generating the metadata. It feels like the API approach is getting lost in the details while the CLI tool is more prominent in this page. Maybe Lisa has some idea on how o adjust the content. 

We can probably add an example of calling the API and storing the response to a file since a file will be what most IDPs expect from an SP and we cover only the raw json response in the API documentation page

Reply:
I agree that the second paragraph is redundant, so I added a commit to put the link in the first paragraph instead. I also agree that if there's a good example we could add to this page (using either of the methods), that'd be great.